src/Billing/ParseERA.php

```diff
@@ -33,9 +33,10 @@ public static function parseERA2100(&$out, $cb)
             // create the 'Claim' service type here.
             //
             if ($GLOBALS['force_claim_balancing']) {
-                $paytotal = $out['amount_approved'];
-                $pattotal = (int)$out['amount_patient'];
-                $adjtotal = $out['amount_charged'] - $paytotal - $pattotal;
+                $chgtotal = floatval($out['amount_charged']);
+                $paytotal = floatval($out['amount_approved']);
+                $pattotal = floatval($out['amount_patient']);
+                $adjtotal = $chgtotal - $paytotal - $pattotal;
```

Comment:
if things ever get confusing (ie. unclear if do int or floatval), i think the following should work `$out['amount_patient'] ?? null`